inference-engine/src/mkldnn_plugin/nodes/mkldnn_def_conv_node.h

```diff
@@ -86,29 +86,82 @@ class MKLDNNDeformableConvolutionNode : public MKLDNNNode {
     InferenceEngine::Precision getRuntimePrecision() const override;
 
 private:
-    size_t group = 1;
-    bool with_bilinear_pad = false;
-    std::vector<ptrdiff_t> stride = {};
-    std::vector<ptrdiff_t> dilation = {};
-    std::vector<ptrdiff_t> paddingL = {};
-
-    int deformable_group = 1;
-
-    jit_def_conv_params jcp = {};
-
-    std::vector<int> sampledCoordsVector;
-    std::vector<float> interpWeightsVector;
+    struct DefConvAttr {
+        size_t group = 1;
+        int deformable_group = 1;
+        bool with_bilinear_pad = false;
+        std::vector<ptrdiff_t> stride = {};
+        std::vector<ptrdiff_t> dilation = {};
+    } defConvAttr;
 
-    std::shared_ptr<jit_uni_def_conv_kernel> def_conv_kernel = nullptr;
+    void prepareParams() override;
+    void updatePadding();
 
-    void prepareSamplingWeights(const std::vector<size_t>& src_strides, const float* offsets, const std::vector<size_t>& off_strides,
-                                const float* modulation = nullptr, const std::vector<size_t>& modulation_strides = {});
-
-    void executeReference(const float* src, const float* weights, float* dst, const std::vector<size_t>& src_strides,
-                          const std::vector<size_t>& wei_strides, const std::vector<size_t>& dst_strides);
-    void executeOptimized(const float* src, const float* weights, float* dst,
-                          const std::vector<size_t>& src_strides, const std::vector<size_t>& dst_strides);
+    std::vector<ptrdiff_t> paddingL = {};
+    std::vector<ptrdiff_t> paddingR = {};
```

Comment:
maybe I missed something but  `paddingR` seems useless in all executors. can we remove it?